canvas_sdk/questionnaires/utils.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+from typing import Any, TypedDict
+
+import yaml
+
+from canvas_sdk.utils.plugins import plugin_only
+
+
+class Response(TypedDict):
+    """A Response of a Questionnaire."""
+
+    name: str
+    code: str
+    code_description: str
+    value: str
+
+
+class Question(TypedDict):
+    """A Question of a Questionnaire."""
+
+    name: str
+    code_system: str
+    code: str
+    code_description: str
+    content: str
+    responses_code_system: str
+    responses_type: str
+    use_in_shx: bool
+    responses: list[Response]
+
+
+class QuestionnaireConfig(TypedDict):
+    """A Questionnaire configuration."""
+
+    name: str
+    use_case_in_charting: str
+    code_system: str
+    code: str
+    can_originate_in_charting: bool
+    search_tags: str
+    scoring_code_system: str
+    scoring_code: str
+    content: str
+    prologue: str
```

Comment:
Do you know if we surface the prologue? I'm not sure I've ever seen this option used.